pkg/deploy-manager/storagecluster.go

```diff
@@ -119,7 +119,7 @@ func DefaultStorageCluster() (*ocsv1.StorageCluster, error) {
 			StorageDeviceSets: []ocsv1.StorageDeviceSet{
 				{
 					Name:     "example-deviceset",
-					Count:    MinOSDsCount,
+					Count:    t.storageClusterConfig.minOSDsCount,
```

Comment:
If it returns a default StorageCluster, shouldn't this be set to default as well?

and for custom Storage Cluster we can have StorageClusterWithOptions(StorageClusterOptions).